ArticleTemplates/liveblogTemplateContributionsEpic.html

```diff
@@ -0,0 +1,8 @@
+<div class="contributions-epic" id="__CONTRIBUTIONS_EPIC_ID__">
+    <style class="contributions-epic__styling">
+        // Contributions Epic styling will go here
```

Comment:
Infact this sits within the Style tags so it'll need to be a CSS comment and the comment on line 6 will need to be an HTML comment.